src/lightning_app/runners/backends/mp_process.py

```diff
@@ -31,7 +31,10 @@ def start(self):
             flow_to_work_delta_queue=self.app.flow_to_work_delta_queues[self.work.name],
             run_executor_cls=self.work._run_executor_cls,
         )
-        self._process = multiprocessing.Process(target=self._work_runner)
+
+        start_method = getattr(self.work, "_start_method", "fork")
```

Comment:
```suggestion
        start_method = self.work._start_method
```
We don't need the getattr, the attribute must exist as per definition in the Work class